js/demo.js

```diff
@@ -0,0 +1,211 @@
+'use strict';
+console.log('app file is connected');
+
+//global variables for grabbing html elements and we are going to set up some arrays to help us hold some values for the logic we need to render random images to the user..
+let totalClicks = 0;
+//pizza objects will live in here and contain the click info.
+const allPizzas = [];
+//store the images we have seen
+let previouslyPickedPizzas = [];
+//grab some html elements
+let pizzaImageSectionTag = document.getElementById('all_pizzas');
+// console.log(totalClicks, allPizzas, previouslyPickedPizza, pizzaImageSectionTag);
+let leftPizzaImage = document.getElementById('left_pizza_img');
+let rightPizzaImage = document.getElementById('right_pizza_img');
+// console.log(leftPizzaImage, rightPizzaImage);
+let leftPizzaOnThePage;
+let rightPizzaOnThePage;
+
+let resultsList = document.getElementById('resultsList');
+let chartResults = document.getElementById('chartResults');
+
+
+
+
+//create a constructor function that will create pizza objects that holds data about our pizzas and the clicking info. {pizza info in these objects.}
+//function express so we a semi colon.....
+const PizzaPicture = function (pizzaName, imageSrc) {
+    // we are assigning our args to the contextual 'this'
+    this.pizzaName = pizzaName;
+    this.imageSrc = imageSrc;
+    //for when they are clicked on we add up by 1
+    this.clicks = 0;
+    // how many did we show the image
+    this.timesShown = 0;
+    allPizzas.push(this);
+};
+
+
+
+// console.log('array? with pizza objects?',allPizzas);
+
+
+
+
+
+
+
+
+
+
+/**
+ * need to create a function to handle what happens when we click on the pizza
+  X -add a check to see if we are clicking on a image
+  X -add to our total clicks to finish rounds
+  X -update the pizza object information times shown... both images each time - X X -views
+  count what image was clicked  - clicks
+  logic -
+  display our random images we are going random number from our pizza objects array
+  we can then update the "src" in the html to update the image that we are seeing on the page
+  add some logic for images that we just used array to store so we dont select those images again for the next round.
+  lastly I want to count total clicks if statement when reach the end of 5 clicks
+  remove the event listener so the user can not click any more.
+
+ *  */
+function handleClickOnPizza(event) {
+    // console.log('from hanlde image click', event.target);
```

Comment:
I know you updated this function, handleClickOnProduct, on your main already but just leaving the note here on a branch.

Edit the click handler to identify which image was clicked

Like this:
function handleClickOnProduct(event) {
// Image clicked
    const clickedImage = event.currentTarget;
    if (clickedImage.id === 'left_product_img') {
// Handle left image click
    } else if (clickedImage.id === 'center_product_img') {
// Handle center image click
    } else if (clickedImage.id === 'right_product_img') {
// Handle right image click
    }
// Anymore click handling logic would go here
}